src/asset-manager/stmatic-withdrawer/StakedTokenWithdrawerSteward.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+import {IERC721} from "openzeppelin-contracts/contracts/token/ERC721/IERC721.sol";
+import {SafeERC20} from "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
+import {OwnableWithGuardian} from "solidity-utils/contracts/access-control/OwnableWithGuardian.sol";
+import {Rescuable721, Rescuable} from "solidity-utils/contracts/utils/Rescuable721.sol";
+import {RescuableBase, IRescuableBase} from "solidity-utils/contracts/utils/RescuableBase.sol";
+import {AaveV3Ethereum} from "aave-address-book/AaveV3Ethereum.sol";
+import {GovernanceV3Ethereum} from "aave-address-book/GovernanceV3Ethereum.sol";
+import {MiscEthereum} from "aave-address-book/MiscEthereum.sol";
+
+import {IStakedTokenWithdrawerSteward} from "./IStakedTokenWithdrawerSteward.sol";
+import {IStMatic} from "./interfaces/IStMatic.sol";
+
+/**
+ * @title StakedTokenWithdrawerSteward
+ * @author TokenLogic
+ * @notice This contract facilitates withdrawals of stMATIC tokens through the Lido staking mechanism
```

Comment:
Please add security considerations and what the purpose of this contract is for.
Mention about rescuable.